lib/cli/commands/listpeers.ts

```diff
@@ -1,11 +1,55 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPeersRequest } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { ListPeersRequest, ListPeersResponse } from '../../proto/xudrpc_pb';
+
+const HEADERS = [
+  colors.red('address'),
+  colors.red('node public key'),
+  colors.red('lnd public keys'),
+  colors.red('inbound'),
+  colors.red('pairs list'),
+  colors.red('xud version'),
+  colors.red('seconds connected'),
+  colors.red('raiden address'),
+];
+
+const createTable = () => {
```

Comment:
I'm noticing this function/code in a couple of your cli-related PRs, maybe make this a cli utils function that accepts the headers as an argument?